src/styles/main-file.scss

```diff
@@ -0,0 +1,21 @@
+@import url("https://fonts.googleapis.com/css2?family=Inter:wght@400;700;800&family=Roboto:wght@400;500;700&display=swap");
+
+@import "./utility/variebles";
```

Comment:
There's a typo in the file pathname. It should be './utility/variables' instead of './utility/variebles'.